pkg/ccl/backupccl/backup_processor.go

```diff
@@ -455,9 +463,28 @@ func runBackupProcessor(
 		todo <- chunk
 	}
 
+	// Passing a nil pacer is effectively a noop if CPU control is disabled.
+	var pacer *admission.Pacer = nil
+	if fileSSTSinkElasticCPUControlEnabled.Get(&clusterSettings.SV) {
+		tenantID, ok := roachpb.ClientTenantFromContext(ctx)
+		if !ok {
+			tenantID = roachpb.SystemTenantID
+		}
+		pacer = flowCtx.Cfg.AdmissionPacerFactory.NewPacer(
+			100*time.Millisecond,
+			admission.WorkInfo{
+				TenantID:        tenantID,
+				Priority:        admissionpb.BulkNormalPri,
+				CreateTime:      timeutil.Now().UnixNano(),
+				BypassAdmission: false,
+			},
+		)
+	}
+	defer pacer.Close()
```

Comment:
might be good to comment that it is safe to close a nil pacer which is why this is outside the if?